Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM golang:1.14.7
-MAINTAINER GitHub, Inc.
+FROM golang:1.20
+LABEL org.opencontainers.image.authors=" GitHub, Inc."
```

Comment:
I don't think it's accurate to say that GitHub is the author here, since this is an open source project and anyone can contribute, and you're contributing here.  It's accurate that GitHub is the maintainer, so let's either back this change out or just remove the maintainer altogether.

Reply:
MAINTAINER is deprecated, with *.authors being the suggested replacement.

I get your point, though! Let me see if i can find a more suitable standard label.

If not, either removal or reverting is fine with me. Between the two, do you have a preference?

Reply:
OK, my thinking is to replace the text of *.authors with the first line of the LICENSE file, and supply the *.url, *.documentation, and *.source URLs, I guess pointing to the Dockerfile, README.md, and repo, respectively.

I'll see about making it a concrete change later (incorporating any feedback you may have by then).

Reference:
https://github.com/opencontainers/image-spec/blob/main/annotations.md

Reply:
(First line of the LICENSE file, to save you the trouble):
> Copyright (c) GitHub, Inc. and LFS Test Server contributors

Reply:
I'd just remove the entry altogether in that case.  I don't think it provides a lot of value in general.